omegaconf/listconfig.py

```diff
@@ -65,6 +65,12 @@ def __init__(
         except Exception as ex:
             format_and_raise(node=None, key=None, value=None, cause=ex, msg=str(ex))
 
+    def _resolve_generic_ref_type(self, ref_type: type) -> type:
+        if ref_type == List:
+            return List[Any]
+        else:
+            return ref_type
+
```

Comment:
why is this not always List[element_type] ?

Reply:
Yeah that would be better, the same goes to Dict. I'll change it.

Reply:
I am not sure this is legal Python, but you get the idea.